pages/linux/setfont.md

```diff
@@ -0,0 +1,20 @@
+# setfont
+
+> Set the terminal font.
+> More information: <https://manned.org/setfont>.
+
+- Change the terminal font:
+
+`setfont {{font}}.gz`
+
+- Change the terminal font by specifying the full filepath:
```

Comment:
```suggestion
- Change the terminal font:
```

Reply:
Having the description be the exact same as the first example feels wrong.

Reply:
```suggestion
- Change the terminal font by a path:
```